packages/calypso-e2e/src/lib/components/editor-settings-sidebar-component.ts

```diff
@@ -133,6 +133,28 @@ export class EditorSettingsSidebarComponent {
 		await expandedLocator.waitFor();
 	}
 
+	/**
+	 * Expands a collapsed section of the sidebar if the section exists in the first place.
+	 * The `Summary` section is no longer collapsible in recent GB iterations
+	 * @see https://github.com/WordPress/gutenberg/commit/201099408131e2abe3cd094f7a1e7e539a350c12
+	 * @deprecated To discourage the adoption of this function
+	 * @todo Remove when all platforms have eventually been migrated
+	 *
+	 * If the section is already open, this method will pass.
+	 *
+	 * @param {string} name Name of section to be expanded.
+	 */
+	async expandSectionIfExists( name: string ): Promise< void > {
```

Comment:
Not saying this approach is wrong, but I'm still curious if you considered adding something like an `expandSummary` method for this specific case? I'm guessing we don't foresee any specific opportunities to reuse `expandSectionIfExists`, and having a dedicated `expandSummary` method might clarify that it can be removed in the future.

Reply:
> but I'm still curious if you considered adding something like an expandSummary method for this specific case?

Sounds more to-the-point. I like it.